main/RFXNames.cpp

```diff
@@ -2759,6 +2760,11 @@ bool GetLightCommand(
 			cmd=Limitless_SetRGBColour;
```

Comment:
Can't we use the 'Set Colour' command ? Yes, we might have to change the 'value' member of the _tLimitlessLights an integer for this